src/app/stripe.service.ts

```diff
@@ -45,13 +42,17 @@ export class StripeService {
       ],
       mode: 'payment',
       currency: donation.currencyCode.toLowerCase(),
-      amount: amountInMinorUnit,
+      amount: this.getAmountInMinorUnit(donation),
       setup_future_usage: 'on_session',
       on_behalf_of: campaign.charity.stripeAccountId,
       paymentMethodCreation: 'manual',
     });
   }
 
+  updateAmount(elements: StripeElements, donation: Donation) {
```

Comment:
Nice find - looks good based on the docblock in StripeElements.update . I couldn't find docs on the stripe website, but I don't think we should need that.

Reply:
It was mentioned in a step inside the finalise on server doc page – which I don't think has a working permalink to the section frustratingly. So it might well be something that's important iff you confirm that way.